src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -179,9 +179,10 @@
 	overflow: hidden;
 	z-index: 1201;
 	min-width: 168px;
+	max-width: 100vw;
 	&.ui-dropdownmenu-options-vertical-margins {
-		margin-top: 6 * @px_base;
-		margin-bottom: 6 * @px_base;
+		margin-top: @dropdown-list-margin;
```

Comment:
do you need this variable for something else? If not, you can remove it.

Reply:
Yes, I am using it to set container max-height.